Libraries/CustomComponents/NavigationExperimental/NavigationHeader.js

```diff
@@ -132,12 +137,22 @@ class NavigationHeader extends React.Component<DefaultProps, Props, any> {
       return props;
     });
 
+    const barHeight = (this.props.statusBarHeight instanceof Animated.Value)
+      ? Animated.add(this.props.statusBarHeight, new Animated.Value(APPBAR_HEIGHT))
```

Comment:
`Animated.add` supports both plain numbers and `Animated.Value`s. So we could just use `Animated.add` instead of this condition.

Though the Flow signature was wrong last time I tried it. And it was throwing error when you pass a number.